ruby/ql/test/query-tests/security/cwe-352/railsapp/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,20 @@
 class ApplicationController < ActionController::Base
+
+  # BAD: `protect_from_forgery` without `with: :exception` can expose an
+  # application to CSRF attacks in some circumstances
+  protect_from_forgery
```

Comment:
It would be good to have an example controller with `protect_from_forgery with: :exception` to confirm that the query doesn't flag that.

Reply:
Added a case of this (with a new action) to the `ArticlesController`.